messages/pt.json

```diff
@@ -1,3 +1,4 @@
 {
-  "store/price.Free": "GRÁTIS"
+  "store/price.Free": "GRÁTIS",
+  "store/price.TBA": "TBA"
```

Comment:
what is TBA?

Reply:
TBA is "To be announced". Since I don't know if it will always be the same text among all countries, I decided to put it on messages.

Reply:
that's so weird

Reply:
I just talked with @davicosta99 and he said the text is going to be different for every country, but since it's not defined yet, it'll stay like that for now.